examples/3d/animated_material.rs

```diff
@@ -25,11 +29,12 @@ fn setup(mut commands: Commands, asset_server: Res<AssetServer>) {
         },
     ));
 
-    let helmet = asset_server.load("models/FlightHelmet/FlightHelmet.gltf#Scene0");
-    for x in -2..3 {
-        for z in -2..3 {
-            commands.spawn(SceneBundle {
-                scene: helmet.clone(),
+    let cube = meshes.add(shape::Cube { size: 0.5 });
+    for x in -1..2 {
+        for z in -1..2 {
```

Comment:
if the materials aren't unique there's no point in spawning more than one helmet

Reply:
just realized this isnt helmets anymore